driver-moab/main/seq_rest_mod.F90

```diff
@@ -736,9 +739,13 @@ subroutine seq_rest_write(EClock_d, seq_SyncClock, infodata, &
     call seq_timemgr_EClockGetData( EClock_d, curr_ymd=curr_ymd, curr_tod=curr_tod)
     call shr_cal_date2ymd(curr_ymd,yy,mm,dd)
     write(year_char,'(i6.4)') yy
-    write(rest_file,"(4a,i2.2,a,i2.2,a,i5.5,a)") &
+    write(rest_file_org,"(4a,i2.2,a,i2.2,a,i5.5,a)") &
          trim(case_name), '.cpl'//trim(tag)//'.r.',trim(adjustl(year_char)),'-',mm,'-',dd,'-',curr_tod,'.nc'
 
+    rest_file = 'mct_'//trim(rest_file_org) ! will actually write here, and return the original name
+    ! moab will write the original one, and read the original one; mct will read the original too
+    ! for the time being, read twice, the same file;; comapre mct_ one with original one to see how different 
```

Comment:
So if we really need MCT output here for comparisons within the MOAB frankencoupler, this should be under MOABCOMP, no?

Reply:
No because MOABCOMP is for inline comparisons.  Writing the file allows us to use cprnc to compare with the other file.

Reply:
So you want to compare MCT restarts from driver MCT and MOAB as well? I am not asking about the MOAB restart file, which is the default now.

Reply:
Turns out although this rest_file name with "mct_" is formed, the file is never written.  Just the moab one.